bin/mail_room

```diff
@@ -2,4 +2,10 @@
 
 require 'mail_room'
 
-MailRoom::CLI.new(ARGV).start
+begin
+  MailRoom::CLI.new(ARGV).start
+rescue => e
```

Comment:
I never want to rescue all errors. It violates the let-it-fail principle that the whole project is built upon. And definitely not as a requirement for everyone. There are people running mail room who's logging solutions can handle the standard error output and this change would probably require some or a lot of work on their part.

Reply:
Yea, I wouldn't want to affect everyone unnecessarily or make it harder for others, but the config idea is a good one. 🙌

Reply:
Is the information I provided below enough to work from?

Reply:
Yea, I think so! I'll push up a new iteration for your review.

Reply:
If this new iteration is acceptable to you, I'll write the tests. thanks!